wurst/_handles/Player.wurst

```diff
@@ -151,6 +151,55 @@ public function player.getTaxRate(player otherPlayer, playerstate whichResource)
 public function player.isRacePrefSet(racepreference preference) returns bool
 	return IsPlayerRacePrefSet(this, preference)
 
+/** Sets the maximum allowed hero amount for a player */
+public function player.setHeroMaxAllowed(int amount)
+	SetPlayerMaxHeroesAllowed(amount, this)
```

Comment:
Also BJ.
But this one is oneliner, probably doesn't matter - if only wurst would automatically inline it.